keras_nlp/models/roberta/roberta_tokenizer.py

```diff
@@ -43,48 +43,39 @@ class RobertaTokenizer(BytePairTokenizer):
     `tf.Tensor` with static shape `[None]`.
 
     Args:
-        vocabulary: string or dict, maps token to integer ids. If it is a
-            string, it should be the file path to a json file.
-        merges: string or list, contains the merge rule. If it is a string,
-            it should be the file path to merge rules. The merge rule file
-            should have one merge rule per line. Every merge rule contains
-            merge entities separated by a space.
+        vocabulary: A dictionary or a string filename path. If passing a
+            filename, the file should be a json file. both json and dictionary
+            should map a single word piece token string to an integer id.
+        merges: A list of merge rules or a string filename path, If passing a
+            filename, the file should have one merge rule per line. Every merge
+            rule contains merge entities separated by a space.
```

Comment:
We could add an example to make it more clear.

Reply:
there is a custom vocabulary example just a few lines later containing an example and passing vocabs and merges to the tokenizer.